bazel-java-deps.bzl

```diff
@@ -24,8 +24,8 @@ def install_java_deps():
             "com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.9",
             "com.fasterxml.jackson.module:jackson-module-scala_2.12:2.9.9",
             "com.github.ben-manes.caffeine:caffeine:2.8.0",
-            "com.github.ghik:silencer-lib_2.12:1.3.1",
-            "com.github.ghik:silencer-plugin_2.12:1.3.1",
+            "com.github.ghik:silencer-lib_2.12.8:1.6.0",
+            "com.github.ghik:silencer-plugin_2.12.8:1.6.0",
```

Comment:
[I believe we're at 2.12.11 now.](https://github.com/digital-asset/daml/pull/5807)

Reply:
Trying out 2.12.11 scala version to see if that works as smoothly as 2.12.8 did on ci.

Reply:
Looks like it's working. Or now @SamirTalwar-DA ?